packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -94,11 +94,13 @@ export class OfficeAddinUsageData {
         this.usageDataOptIn();
       }
 
-      appInsights.setup(this.options.instrumentationKey)
-        .setAutoCollectExceptions(false)
-        .start();
-      this.usageDataClient = appInsights.defaultClient;
-      this.removeApplicationInsightsSensitiveInformation();
+      if (this.options.usageDataLevel === "on") {
```

Comment:
It would be better if there is an enum for the usageDataLevel values. Is this not a boolean because there is more than just "on" and "off"? 

Please ensure the value is normalized to be lowercase or convert it to lowercase for this comparison.

Reply:
Yeah we actually can use the UsageDataLevel enum. Will fix this